test/cmdlineTests.sh

```diff
@@ -28,27 +28,86 @@
 
 set -e
 
-REPO_ROOT="$(dirname "$0")"/..
+REPO_ROOT=$(cd "$REPO_ROOT" && pwd)
```

Comment:
`$(...)` executes the command(s) inside the parantheses and returns the output of that command as a value. `&&` just means "only execute the next command if the previous command succeeded" i.e. exactly as the `&&` in C. `pwd` prints the (absolute) path to the current directory.